tips/models.py

```diff
@@ -20,6 +21,10 @@ class TipOfDay(models.Model):
     active = models.BooleanField(default=True)
     seen_by = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name='seen_tips', blank=True)
     groups = models.ManyToManyField(Group, related_name='tips', blank=True)
+    for_staff = models.BooleanField(default=True, help_text='This tip is intended for staff')
+    for_nonstaff = models.BooleanField(default=False,
+                                       help_text='This tip is intended for non_staff users')
+    sites = models.ManyToManyField(Site, blank=True)
```

Comment:
Možno proidať help text, že keď sa nezadá žiadna site, zobrazuje sa to na všetkých?